tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -111,46 +113,99 @@ type stressConfig struct {
 }
 
 // NewStresser creates stresser from a comma separated list of stresser types.
-func NewStresser(s string, sc *stressConfig, m *member) Stresser {
+func NewStresser(s string, sc *stressConfig, members []*member) Stresser {
```

Comment:
That's not good. etcd-runner should be able to spin up an arbitrary number of runners without conflicts.